config.js

```diff
@@ -1167,6 +1167,7 @@ var config = {
      forceJVB121Ratio
      forceTurnRelay
      hiddenDomain
+     hiddenFromRecorder
```

Comment:
1. Shouldn't we name this differently, maybe something like hiddenFromRecorderEnabled or  hidingFromRecorderEnabled?
2. Also why it is listed in the undocumented options?
3. I'm wondering if it will be useful to add it in the list for overriding? WDYT? The use case I'm thinking of is to generally keep it `false` and set it to true for a specific jibri maybe?

Reply:
Nope, this should not be overridable to be able to protect deployments where the server-side protection is not running [mod_presence_identity.lua](https://github.com/jitsi/jitsi-meet/blob/master/resources/prosody-plugins/mod_presence_identity.lua)